fabric-item-api-v1/src/testmod/java/net/fabricmc/fabric/test/item/CustomDamageTest.java

```diff
@@ -55,5 +55,16 @@ public Text getName(ItemStack stack) {
 			int v = stack.getOrCreateNbt().getInt("weird");
 			return super.getName(stack).copy().append(" (Weird Value: " + v + ")");
 		}
+
+		@Override
+		public ItemStack getRecipeRemainder(ItemStack stack) {
+			if (stack.getDamage() < stack.getMaxDamage()) {
+				ItemStack moreDamaged = stack.copy();
```

Comment:
```suggestion
				ItemStack moreDamaged = stack.copy();
				moreDamaged.setCount(1);
```

Probably overkill but I suppose this would be a "best practice".